src/tito/builder/main.py

```diff
@@ -79,7 +79,12 @@ def __init__(self, name=None, build_dir=None,
 
         self.quiet = self._get_optional_arg(kwargs, 'quiet', False)
         self.verbose = self._get_optional_arg(kwargs, 'verbose', False)
-        self.fetch_sources = self._get_optional_arg(kwargs, 'fetch_sources', False)
+
+        # The default for the builder instantiated from Releasers
+        config_fetch_default = False
+        if self.config.has_option(BUILDCONFIG_SECTION, "fetch_sources"):
+            config_fetch_default = self.config.get(BUILDCONFIG_SECTION, "fetch_sources")
+        self.fetch_sources = self._get_optional_arg(kwargs, 'fetch_sources', config_fetch_default)
```

Comment:
Not your fault in the slightest, just pointing out something that annoys me - Ten minutes ago I wrote something similar in another PR in `MockBuilder`. It's awful, we can't parse the config and deal with argument defaults in a thousand different places.

We need a to unify it and properly set defaults from the config file for all recognized arguments.

Reply:
Yes, that's a reasonable TODO item.  I didn't want to change too many things in this PR, but I was tempted to do this right away :-)

Reply:
Me too but I was scared to do it without thorough testing :-)